network-api/networkapi/wagtailpages/templatetags/localization.py

```diff
@@ -17,7 +17,7 @@
     'fy-NL': 'fy_NL',
     'nl': 'nl_NL',
     'pl': 'pl_PL',
-    'pt': 'pt_BR',  # our main focus is Brazilian Portuguese
+    'pt-BR': 'pt_BR',  # our main focus is Brazilian Portuguese
```

Comment:
we probably don't need this comment anymore, since there's no mismatch now.

Reply:
```suggestion
    'pt-BR': 'pt_BR',
```